src/shared/models/chat.model.ts

```diff
@@ -0,0 +1,7 @@
+import type * as mongoose from 'mongoose';
+export interface Chat {
+  _id?:{type: mongoose.Types.ObjectId},
```

Comment:
we can just use a string for _id now. we can look at the id as a string in the server

Reply:
Testing

Reply:
Good eye thanks for the catch. I refactored.